index.bs

```diff
@@ -501,21 +502,18 @@ at zero, increments, and is reset — is a good example of a privacy friendly
 temporary identifier.
 
 <h3 class=question id="first-third-party">
-  How does this specification distinguish between behavior in first-party and
-  third-party contexts?
+  How does this specification protect end users from storing persistent data 
+  on a user’s device across browsing session?
```

Comment:
While I think your new question may be one worth asking in the questionnaire, I'd rather it be spun off into a separate PR that *adds* this question, instead of one which *replaces* the first-/third-party question. Among other things, this new question is significantly narrower than the original.

Reply:
Maybe pursue removing or modifying the first-/third-party question in a separate PR as well.

Reply:
There are two key changes in this pull request.

1.	Recognising the full range of parties involved in the web.
2.	Resolving first and third-party bias.

Before splitting the second change into another pull request I’m keen to gauge wider views on this and have been encouraged to raise these issues via specific text changes. 

If smaller participants on the web are denied the ability to “band together” to deliver services that only larger participants can deliver due to their scale, then we have to consider the W3C’s antitrust policy. What work has been done on this document to verify it does not unconsciously steer specifications towards this outcome? Whilst I recognise that the subject of competition and antitrust may not be one other people feel willing or able to engage with that does not resolve the matter.

There are other considerations related to people’s trust choices and trusting supply chains. The document, and reviewers of issues, have categorically stated that people can not trust supply chains. I’ve not seen the evidence for this from any authoritative sources. As a minimum the document should reference such sources for new readers like myself.  Perhaps approaching the problem from the perspective of the conditions under which a person can or can not trust a supply chain would add a much needed dimension to the document?

Reply:
I don't think this is the right forum to have that debate. We're trying to craft some guidelines here, not open up a debate about the nature of the web. Can I suggest that we go with Tess's suggestion and split this out into a new question?

Reply:
Following discussion in the IWA BG where there was broad support to discuss the subject further, and considering the opportunity at TPAC to hold a debate on these matters, I have [proposed a session for October](https://github.com/w3c/web-advertising/issues/87). The output of that session will help inform the future of these definitions and how they apply to the questionairre and the work of the TAG and PING.

It is clear that the current definitions or interpretations of parties is too simplistic to solve the myriad of challanges we are all working on.

Reply:
A [TPAC break session proposal](https://github.com/w3c/web-advertising/issues/87#issuecomment-700806304) has been added following the Improving Web Advertising Business Group today and discussion on the session there.